rosbridge_server/launch/rosbridge_websocket_client.launch

```diff
@@ -17,7 +17,7 @@
   <group unless="$(arg ssl)">
     <node name="rosbridge_websocket_client" pkg="rosbridge_server" type="rosbridge_websocket_client" output="screen">
       <param name="authenticate" value="$(arg authenticate)" />
-      <param name="server_address" value="$(arg url)"/>
+      <param name="server_uri" value="$(arg uri)"/>
```

Comment:
I mean, it wasn't changed at the top of the launch file where the argument is defined.